### PR TITLE
Documentation for new TCP Lighting (Greenwave Reality) component

### PR DESCRIPTION
**Pull request by dfiel**
**Description:**
Documentation for the TCP Lighting component (light.tcpbulbs

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#11282

## Checklist:

  - [x] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`. 
  - [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/light.tcpbulbs.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+```yaml
+light:
+  - platform: tcpbulbs
+    host: XXX.XXX.XXX.XXX
+```
+
+Configuration variables:
+
+- **host** (*Required*): The IP Address for the TCP Connected Gateway.
```
Comment (frenck) [inline source/_components/light.tcpbulbs.markdown]: Please use the configuration tags in order to specify configuration variables in the documentation. For more information see:
https://home-assistant.io/developers/documentation/create_page/#configuration

File: source/_components/light.tcpbulbs.markdown [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+layout: page
+title: TCP Connected Light
+description: "Instructions on how to set up TCP Connected lights within Home Assistant."
+date: 2017-11-17 18:50
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Light
+ha_release: 0.XX.X
```
Comment (frenck) [inline source/_components/light.tcpbulbs.markdown]: Please set it to `0.59` to start with.

Comment (dfiel) [reply source/_components/light.tcpbulbs.markdown]: @frenck Fixed.

File: source/_components/light.tcpbulbs.markdown (hunk 1 above)
Comment (dfiel) [reply source/_components/light.tcpbulbs.markdown]: @frenck Fixed.

File: source/_components/light.tcpbulbs.markdown [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+
+To configure the connection to the gateway, add the following to your `configuration.yaml` file:
+
+```yaml
+light:
+  - platform: tcpbulbs
+    host: XXX.XXX.XXX.XXX
+```
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/light.tcpbulbs.markdown]: Please remove this header as well. It will be provided by the configuration tags you've added earlier.

Comment (dfiel) [reply source/_components/light.tcpbulbs.markdown]: @frenck Fixed!

**Comment by frenck** [comment]
@dfiel, thank you for making those improvements!
PR is ready to merge and can be merged as soon as the parent PR gets merged.

**Comment by dfiel** [comment]
@frenck The main pull request was merged, can this be merged as well? Thanks.

**Comment by frenck** [comment]
@dfiel Thanks for the ping. Nevertheless, a second parent PR was opened?
Will these docs change?  I would suggest opening a secondary PR on the documentation for that.

**Comment by dfiel** [comment]
@frenck No documentation changes with the new PR. Thanks for merging!

**Pull request merged**